CMakeLists.txt

```diff
@@ -6,11 +6,13 @@ catkin_simple()
 
 include(ExternalProject)
 
+set(YAML_CPP_TAG 11607eb5bf1258641d80f7051e7cf09e317b4746)
```

Comment:
Sure, will add a comment. I just wanted to get rid of those std::auto_ptr compiler warnings.